pinot-core/src/test/java/org/apache/pinot/queries/InterSegmentAggregationSingleValueQueriesTest.java

```diff
@@ -568,4 +568,32 @@ public void testNumGroupsLimit() {
             InstancePlanMakerImplV2.DEFAULT_GROUPBY_TRIM_THRESHOLD));
     assertTrue(brokerResponse.isNumGroupsLimitReached());
   }
+
+  @Test
+  public void testDistinctSum() {
+    String query = "select DISTINCTSUM(column1) as v1, DISTINCTSUM(column3) as v2 from testTable";
+
+    // Without filter, query should be answered by DictionaryBasedAggregationOperator (numEntriesScannedPostFilter = 0)
```

Comment:
Can you add a test for EXPLAIN PLAN as well using this function especially for the dictionary based case ?